src/snyk/common/constants/globalState.ts

```diff
@@ -1,3 +1,4 @@
 export const MEMENTO_FIRST_INSTALL_DATE_KEY = 'snyk.firstInstallDate';
+export const MEMENTO_ANONYMOUS_ID = 'snyk.anonymousId';
```

Comment:
Ah, going the Jetbrains route? ;)

Reply:
Absolutely, it makes sense to be aligned on that 👍